doc/conseil.sql

```diff
@@ -391,112 +362,104 @@ ALTER TABLE ONLY public.operations
 
 
 --
--- TOC entry 2064 (class 1259 OID 7467561)
 -- Name: fki_block; Type: INDEX; Schema: public; Owner: -
 --
 
 CREATE INDEX fki_block ON public.operation_groups USING btree (block_id);
 
-CREATE INDEX ix_operation_groups_block_level ON public.operation_groups USING btree (block_level);
```

Comment:
Note, this simply moved down in the file for some reason. I did not, in fact, remove an index.